main/src/ca/uwaterloo/flix/language/phase/Simplifier.scala

```diff
@@ -245,6 +176,50 @@ object Simplifier {
         SimplifiedAst.JvmMethod(ident, fparams, exp, retTpe, simplifyEffect(eff), loc)
     }
 
+    /**
+      * Translates the given `ApplyAtomic` expression to the corresponding [[SimplifiedAst]] expression.
+      */
+    def visitAtomic(atomicExp: LoweredAst.Expression.ApplyAtomic): SimplifiedAst.Expression.ApplyAtomic = {
```

Comment:
We usually don't do this. Can we inline this instead?

Reply:
Its a function thats only called once, from one place.

Reply:
Sure :)